src/measure_theory/bochner_integration.lean

```diff
@@ -1033,8 +1058,13 @@ begin
   { rw [integral_undef hf, norm_zero], exact to_real_nonneg }
 end
 
+lemma ennnorm_integral_le_lintegral_norm (f : α → E) :
+  (nnnorm (∫ a, f a ∂μ) : ennreal) ≤ ∫⁻ a, (ennreal.of_real ∥f a∥) ∂μ :=
```

Comment:
Add a version with `nnnorm (f a)` in the RHS? It's the goal after the first `rw` and it looks more natural to me (same formula for `ennnorm` in both sides).

Reply:
I struggled a bit, since I don't know whether we have a "normal form" between `ennreal.of_real ∥x∥` and `(nnnorm x : ennreal)`, so I wasn't very consistent myself. Do you have a preference of what should be the normal form? Then I can use that on both sides.

Reply:
I like `coe (nnnorm x)` because it can be `norm_cast`ed to `nnreal`.